fighthealthinsurance/utils.py

```
@@ -608,3 +610,19 @@ def generate_random_unsupported_filename() -> str:
     """Generate a random short filename with .unsupported extension."""
     rand_str = "".join(random.choices(string.ascii_lowercase + string.digits, k=8))
     return rand_str + ".unsupported"
+
+
+async def _try_pandoc_engines(command: list[str]):
+    engines = [None, "pdflatex", "xelatex", "laulatex", "wkhtmltopdf", "weasyprint"]
+    if "--sandbox" not in command:
+        command = command + ["--sandbox"]
```

Ruff (0.14.8)

Consider `[*command, "--sandbox"]` instead of concatenation

Replace with `[*command, "--sandbox"]`

(RUF005)